.vscode/cspell.json

```diff
@@ -85,28 +104,42 @@
   "overrides": [
     {
       "filename": "**/eng/pipelines/templates/jobs/archetype-sdk-client.yml",
-      "words": ["lcov", "iname"]
+      "words": [
```

Comment:
Is it worth keeping these overrides for each file or should we just add them to the word list?